buku

```diff
@@ -6026,10 +6030,6 @@ POSITIONAL ARGUMENTS:
     -o, --open [...]     browse bookmarks by indices and ranges
                          open a random bookmark, if no arguments
     --oa                 browse all search results immediately
-    --default-scheme S   if scheme is missing from uri, assume S.
-                         valid values are http, https (default http)
-                         (override with environment variable
-                         BUKU_DEFAULT_SCHEME)
```

Comment:
You'll still want the param to be _mentioned_ in the `--help` output at least :sweat_smile: